test/FunctionalTests/Tests/DataOData/Tests/OData.Writer.Tests/ConvertToUriLiteralTests.cs

```diff
@@ -484,6 +485,21 @@ public void ConvertToUriLiteralCollectionTest()
             this.RunTestCases(testCases);
         }
 
+
+        [TestMethod]
+        public void ConvertToUriLiteralWhenIeee754CompatibleSetTrue()
+        {
+            string expectedWhenieee754ParamSetFalse = "[-9223372036854775808,9223372036854775807]";
+            string expectedWhenieee754ParamSetTrue = "[\"-9223372036854775808\",\"9223372036854775807\"]";
+
+            var parameter = new ODataCollectionValue() { TypeName = EntityModelUtils.GetCollectionTypeName("Edm.Int64"), Items = new object[] { Int64.MinValue, Int64.MaxValue } };

```

Comment:
```suggestion
            ODataCollectionValue parameter = new ODataCollectionValue
            {
                TypeName = EntityModelUtils.GetCollectionTypeName("Edm.Int64"),
                Items = new object[] { Int64.MinValue, Int64.MaxValue }
            };
```
Style here prefers types over vars
Some newlines also adds some visibility.